Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/NewMessage/newMessage.tsx

```diff
@@ -503,14 +503,20 @@ export default class NewMessage extends React.Component<INewMessageProps, formSt
 
     private onImageLinkChanged = (event: any) => {
         let url = event.target.value.toLowerCase();
-        if (!(url.startsWith("https://") || (url.startsWith("data:image/png;base64")) || (url.startsWith("data:image/jpeg;base64")) || (url.startsWith("data:image/gif;base64")))) {
-            this.setState({
-                errorImageUrlMessage: "URL must start with https://"
-            });
-        } else {
+        if (url === "") {
```

Comment:
Rather than having a new block, let's put this check in the check with the startsWith
if (!(url === "" || (url.startsWith...))

Reply:
done